src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -111,11 +110,15 @@ public Api getApi() {
      * @return Docker container IDs.
      */
     public synchronized @Nonnull Set<String> getContainerIDs() {
-        return (containerIDs == null) ? new HashSet<String>() : new TreeSet<String>(containerIDs);
+        return (containerIDs == null) ?  Collections.<String>emptySet() : new TreeSet<String>(containerIDs);
```

Comment:
it would be better IMHO to always initialise containerIDs to a set to avoid multiple checks like this.